docs/content/setup/integrations.md

```diff
@@ -59,3 +59,116 @@ them to OpenFGA queries.
 
 !!! info "Third Party Solutions"
     A third-party example of such a webhook would be Platform Mesh's [rebac-authz-webhook](https://github.com/platform-mesh/rebac-authz-webhook).
+
+## Lima
+You can run kcp inside a [Lima](https://github.com/lima-vm/lima)-managed VM, which makes it portable across macOS, Linux, and Windows (via WSL2). This setup gives you a disposable kcp control plane that integrates smoothly with your host kubectl.
```

Comment:
Can we add a minimal README on what this integration does? 

This is essentially a playground/dev environment, where one can start a single instance of kcp (kcp in the box) for testing or limited-scope use cases. And I would like to note that this is in no way intended for production usage so we dont send wrong signal for the reader.